22_pipeslab/pipeslab.txt

```diff
@@ -3,19 +3,106 @@ For each of the following, type a single command line that performs:
   1. Print the number of lines from "ls -l" which represent a path that has execute permission to their owner.
   Answer: 
 
+maxim@maxim-virtual-machine:~$ ls -l | grep '^...x' | cut -d' ' -f3,9
```

Comment:
what's clever! but you need to count the lines.. there's a nice command called ``wc`` (Word Count):
```
wc -l # count lines
wc -c # count characters
wc -w # count words
```